### DISCUSSION TITLE
Validate Json Response by the negative

### DISCUSSION THREAD

**Discussion by axelbarfod** [pactumjs/pactum#239] [Q&A] [answered]
Hey I'm new to using the framework and I have a scenario which I would like to validate, in which I would to certain fields don't show up on there response. I know there is a .JsonExpect() which I can specify the fields I want to validate, is there a way to validate the oposite?

**Comment by ASaiAnudeep** [comment]
There is no ideal way to validate opposites. But [notIncludes](https://pactumjs.github.io/api/matching/notIncludes.html) matcher would help to perform these kind of assertions to some extent.

**Comment by axelbarfod** [comment, marked as answer]
Tried using the [notIncludes ](https://pactumjs.github.io/api/matching/notIncludes.html#examples)but I keep getting the error: 

 `AssertionError [ERR_ASSERTION]: Json doesn't have a "object" at "$.sendBirdAccessToken"`

Here is my sample test: 

```
await spec()
            .get(base_url + /users/123`)
            .expectJsonMatch({
                'sendBirdAccessToken': notIncludes('sendBirdAccessToken')
});

```
This is the response from my API:

`{
    "id": "someid",
    "username": "ovz1wot3e",
    "email": "raraara+ovz1wot3e@gmail.com",
    "dateOfBirth": null,
    "sendBirdAccessToken": "token",
    "products": [],
    "myReviews": [],
    "kyc": null,
    "deliveryAddresses": [
    ]
}`

I want to validate that the sendBirdAccessToken isn't sent on the response.

**Reply by ASaiAnudeep** [reply]
As you are making assertion on the root of the response object, the assertion should look like

```js
await spec()
  .get(base_url + /users/123`)
  .expectJsonMatch(notIncludes('sendBirdAccessToken'));
```

**Reply by axelbarfod** [reply]
Thanks worked perfectly!
